calico-cloud_versioned_docs/version-3.17/image-assurance/overview.mdx

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
@penkeysuresh could you clarify why we need v3.17 docs? That hasn't been released yet.

Reply:
I was following the request, could you clarify this @bmckercher123 ?